content/docs/command-reference/version.md

```diff
@@ -36,36 +34,32 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: a stable version of DVC (for example `1.7.4`), that can
+  be [installed](/doc/install) using the official package stored in
+  [Python Packaging Authority](https://www.pypa.io/) or with a binary. These
+  releases are tagged in the DVC repository (see
+  [changelog](https://github.com/iterative/dvc/releases) for release history).
 
-  > 💡 To only get this number, use `dvc -V`.
+  If there are any issues reported with the official build, they can be traced
+  using the official version number.
 
-- **Development version**: `pip install git+git://github.com/iterative/dvc` will
-  install DVC using the `master` branch of DVC's repository. Another way of
-  setting up the development version is to clone the repository and run
-  `pip install .`.
+- **Development version**: contains all the changes done to the `master` branch
+  of DVC's repository, following the latest official release (for example
+  `1.7.4+292cab.mod`) . It can be
```

Comment:
> contains all the changes ... (for example `1.7.4+292cab.mod`)

That parenthesis no longer makes sense. Is that an example of the changes contained in the dev version? Please, when rewriting sentences don't leave old chunks there without making sure it's still readable. Feel free to make more sentences to keep that other important info (the dev version number example) if needed.

Reply:
But in fact you can just remove it in this case, as mentioned in https://github.com/iterative/dvc.org/pull/1814#pullrequestreview-505852333

Reply:
I'll do it so we can merge this now:

```suggestion
- **Development version**: contains all the changes done to the `master` branch
  of DVC's repository, following the latest official release. It can be
```